src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
+  margin: 0 auto;
+
+  max-width: 1200px;
+  min-width: 800px;
+  --blue-color: #00acdc;
+  --header-height: 23px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__link {
+  max-height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  margin: var(--header-height) 0;
+  padding-left: 0;
+  font-size: 0;
+}
+
+.nav__item {
+  display: inline-block;
   margin: 0;
+  padding: 0;
+  align-items: center;
+  height: fit-content;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  transition: 0.2s;
+  padding: var(--header-height) 0;
+  height: 60px;
```

Comment:
Overall - this works, but you are Some minor issues here

- on `nav__link`: `<a>` has `display: inline` by default, sho `height` is ignored
- you are setting `line-height` and `font-size` to `nav__item` while actually they are needed for `nav__link`. You even duplicate `font-size` for some reason
- `align-items` does not have effect when display is set to `inline-block`

Take some time to clean up how `nav__item` and `nav__link` are implemented

Reply:
wow, this was helpful! thank you!